recipes/trust4/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: trust4
+  version: 1.0.2.beta
```

Comment:
Beta versions aren't meant for end-users, please ping us when there's a final release.